compiler/ballerina-lang/src/main/resources/grammar/BallerinaParser.g4

```diff
@@ -58,6 +58,10 @@ blockFunctionBody
     :   LEFT_BRACE statement* (workerDeclaration+ statement*)? RIGHT_BRACE
     ;
 
+blockStatement
```

Comment:
Are we only just adding the block statement? What about the statements which make use of block statements? Or will that be fixed with the new parser?

Reply:
this is fixed with new parser